vcbuild.bat

```diff
@@ -571,6 +572,11 @@ rem ***************
 rem   Subroutines
 rem ***************
 
+:validatepython
+echo Looking for Python 2.x
+call :run-python --version > nul 2> nul
+if errorlevel 1 echo Could not find Python 2.x installation && exit /b 1
+
```

Comment:
1. `:run-python` will echo `Could not find python2` so IMHO no need to echo again. (L561)
2. should end subroutine with either `exit /b` or `goto :EOF`

Reply:
Agree, let's not duplicate the validation.

Reply:
`:run-python` is very verbose, especially if no Python is found (see https://github.com/nodejs/node/issues/16864). Stdout and stderr is redirected to `nul`, to keep it clean.

Reply:
Then maybe `:run-python` should be fixed. I was planning to do it myself, I think it's a simple error in the code.